src/jsx-runtime/index.ts

```diff
@@ -118,7 +117,7 @@ jsx.renderDOM = (
         }
       }
       // @ts-ignore
-      elem[name.toLowerCase()] = attrVal
```

Comment:
any reason why lowercase has been removed?
this was done on purpose to support events in camelCase like `onClick`, which only exist in lowecase in JS (`onclick`, etc.)

Reply:
having it lowercase breaks `className` as it has to be lowerCamelCase:

![afbeelding](https://user-images.githubusercontent.com/1548276/120103182-b8fb0300-c14e-11eb-8439-c75f7f9a616c.png)

But I guess there could be a dictionary or something to convert to the case expected by the browsers? Or maybe even keep either permutation and just have it decided itself what sticks?

Reply:
maybe we should lowercase only in case the `name` starts with `on`, otherwise className will become classname.

anyway, I'd rather use official DOM property names to be honest. so it's ok to keep this change. We will need to document somewhere that things like `onClick` arent supported, like in React, but it should be `onclick` `onchange` etc